File: CSharp/Math/RandomRange.cs
using System;
using static System.Console;
					
public class Program {
	public static void Main() {
		var rand = new Random();
		for (int i = 0; i < 10; i++) {
			WriteLine(rand.Next(10000, 99999));
		}
		WriteLine();
		for (int i = 0; i < 10; i++) {
			rand = new Random();
			WriteLine(rand.Next(10000, 99999));
		}
	}
}

//https://pt.stackoverflow.com/q/55701/101

File: C/Math/RandomRange.c
#include <stdio.h>
#include <time.h>

int main(void) {
	srand(time(NULL));
	for (int i = 0; i < 100; i++) printf("%i\n", 1001 + rand() % 4000);
}

//https://pt.stackoverflow.com/q/39215/101
